pkg/api/vlabs/validate.go

```diff
@@ -274,6 +277,9 @@ func (a *Properties) validateOrchestratorProfile(isUpdate bool) error {
 						return errors.Errorf("loadBalancerSku is only available in Kubernetes version %s or greater; unable to validate for Kubernetes version %s",
 							minVersion.String(), o.OrchestratorVersion)
 					}
+					if *a.OrchestratorProfile.KubernetesConfig.ExcludeMasterFromStandardLB == false {
```

Comment:
let's use the helper function `helpers.IsFalseBoolPointer` here instead of dereferencing the pointer which might be nil